maat-orchestration/src/test/java/uk/gov/justice/laa/crime/orchestration/mapper/MeansAssessmentMapperTest.java

```diff
@@ -43,7 +43,56 @@ void givenApiFindHardshipResponseWithSection_whenFindHardshipResponseToHardshipD
         softly.assertThat(actual)
                 .usingRecursiveComparison()
                 .ignoringCollectionOrder()
+                .ignoringFields("incomeEvidence.applicantIncomeEvidenceList",
```

Comment:
The name of this method appears to be inaccurate, can we take the opportunity to fix this please?

Reply:
Fixed.